website/src/documentation/transforms/python/element-wise/kvswap.md

```diff
@@ -19,21 +19,64 @@ limitations under the License.
 -->
 
 # Kvswap
-<table align="left">
-    <a target="_blank" class="button"
+
+<script type="text/javascript">
+localStorage.setItem('language', 'language-py')
+</script>
+
+<table>
+  <td>
+    <a class="button" target="_blank"
         href="https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.util.html#apache_beam.transforms.util.KvSwap">
-      <img src="https://beam.apache.org/images/logos/sdks/python.png" width="20px" height="20px"
-           alt="Pydoc" />
-     Pydoc
+      <img src="https://beam.apache.org/images/logos/sdks/python.png"
+          width="20px" height="20px" alt="Pydoc" />
+      Pydoc
     </a>
+  </td>
 </table>
 <br>
-Takes a collection of key-value pairs and returns a collection of key-value pairs 
+
+Takes a collection of key-value pairs and returns a collection of key-value pairs
 which has each key and value swapped.
 
-## Examples
-See [BEAM-7389](https://issues.apache.org/jira/browse/BEAM-7389) for updates. 
+## Examplse
+
+In the following example, we create a pipeline with a `PCollection` of key-value pairs.
+Then, we apply `Keys` to extract the keys and discarding the values.
```

Comment:
"Then, we apply `KvSwap` to swap the keys and values."

Reply:
Good old copy paste :)

Fixed